lib/domain/entity/my_page_info.dart

```diff
@@ -0,0 +1,11 @@
+class MyPageInfo {
+  final String name;
+  final List<String> tags; // part와 club을 합친 태그 리스트
```

Comment:
만약 해당 데이터가 중요한 주석인 경우에는 ///를 활용해서 type hint를 줄 수도 있어요
```suggestion
  /// part와 club을 합친 태그 리스트
  final List<String> tags; 
```